website/pages/docs/deployment/ecs.md

```diff
@@ -51,24 +51,8 @@ In order to inject the config file into the prebuilt container you will have to
 cat cloudquery.yml | base64
 ```
 
-## Step 2: Create an ECS Cluster
-The first step in deploying CloudQuery on AWS ECS is to create an ECS cluster. To create an ECS cluster, use the following command:
-
-Prior to running replace `<REPLACE_WITH_ECS_CLUSTER_NAME>` with the name you want to give to your ECS cluster.
-```bash
-aws ecs create-cluster --cluster-name <REPLACE_WITH_ECS_CLUSTER_NAME>
-```
 
-
-## Step 3: Create a Log Group
-The next step is to create a log group for your ECS task. To create a log group, use the following command:
-
-Prior to running replace `<REPLACE_WITH_LOG_GROUP_NAME>` with the name you want to give to your log group.
-```bash
-aws logs create-log-group --log-group-name <REPLACE_WITH_LOG_GROUP_NAME>
-```
-
-## Step 4: Create Secret to CloudQuery API Key
+## Step 2: Create Secret to CloudQuery API Key
```

Comment:
```suggestion
## Step 2: Create a Secret for the CloudQuery API Key
```

Reply:
Btw in the previous section (sorry couldn't add a comment) there's no Windows version for the base64 encode. Apparently it's:

```
echo "your_string" | certutil -encodehex -f - 0
```

Reply:
This tutorial has been up for over a year and people have not asked about that. If we want to add windows support I think that can be done in a different PR